numpy/random/tests/test_generator_mt19937.py

```diff
@@ -568,7 +568,7 @@ def test_choice_nonuniform_replace(self):
     def test_choice_uniform_noreplace(self):
         random = Generator(MT19937(self.seed))
         actual = random.choice(4, 3, replace=False)
-        desired = np.array([0, 1, 3], dtype=np.int64)
+        desired = np.array([2, 0, 3], dtype=np.int64)
```

Comment:
This would be a good place to add a test that shows the use of the `shuffle` kwarg

Reply:
added a test, thanks.